tests/image/test_backbones.py

```diff
@@ -34,10 +35,10 @@ def test_image_classifier_backbones_registry(backbone, expected_num_features):
 
 @pytest.mark.parametrize(["backbone", "pretrained", "expected_num_features"], [
     pytest.param(
-        "resnet50", "supervised", 2048, marks=pytest.mark.skipif(not _TORCHVISION_AVAILABLE, reason="No torchvision")
+        "resnet50", 'supervised', 2048, marks=pytest.mark.skipif(not _TORCHVISION_AVAILABLE, reason="No torchvision")
```

Comment:
@ethanwharris do you want to use Black here too?

Reply:
@Borda Yes I think we will switch to black. Want to get a few PRs in first though to try to avoid some conflicts :smiley: